openupgrade_scripts/scripts/snailmail/15.0.0.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,11 @@
+---Models in module 'snailmail'---
+---Fields in module 'snailmail'---
+snailmail    / mail.notification        / failure_type (False)          : selection_keys is now '['mail_email_invalid', 'mail_email_missing', 'mail_smtp', 'sms_acc', 'sms_credit', 'sms_number_format', 'sms_number_missing', 'sms_server', 'sn_credit', 'sn_error', 'sn_fields', 'sn_format', 'sn_price', 'sn_trial', 'unknown']' ('['BOUNCE', 'RECIPIENT', 'SMTP', 'UNKNOWN', 'sms_acc', 'sms_credit', 'sms_number_format', 'sms_number_missing', 'sms_server', 'sn_credit', 'sn_error', 'sn_fields', 'sn_format', 'sn_price', 'sn_trial']')
```

Comment:
Do we handle the changed selection keys elsewhere?

Reply:
Yes, in this module, Selection pairs are not changed so I don't have migration for this module.